docs/root/intro/version_history.rst

```diff
@@ -30,6 +30,10 @@ Version history
   value to override the default HTTP to gRPC status mapping.
 * http: no longer close the TCP connection when a HTTP/1 request is retried due
   to a response with empty body.
+* listeners: all listener filters are now governed by the :ref:`listener_filters_timeout
+  <envoy_api_field_Listener.listener_filters_timeout>` setting. The hard coded 15s timeout in
+  the :ref:`TLS inspector listener filter <config_listener_filters_tls_inspector>` is superseded by
```

Comment:
The prior TLS inspector listener filter _timeout_ is replaced, not the TLS inspector listener, right?

Reply:
It reads: "The hard coded 15s timeout in the TLS inspector listener filter is superseded by..." I think that is OK?